ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml

```diff
@@ -68,6 +68,8 @@ periodics:
       command:
       - /osde2e
       env:
+      - name: POLLING_TIMEOUT
+        value: "7000"
```

Comment:
This isn't our job, i doubt you need to be changing it

Reply:
Hi Mike, currently the default polling timeout is set to 3600 seconds, we need around 6000 seconds for our installation + tests to complete. This aims to increase the default timeout.